arlo-client/src/components/CreateAudit.test.tsx

```diff
@@ -86,15 +104,20 @@ describe('CreateAudit', () => {
       expect(apiMock).toBeCalledTimes(1)
       expect(apiMock).toHaveBeenNthCalledWith(1, '/auth/me')
     })
-
+    fireEvent.change(getByLabelText('Give your new audit a unique name.'), {
+      target: { value: 'Audit Name' },
+    })
     fireEvent.click(getByText('Create a New Audit'), { bubbles: true })
 
     await wait(() => {
       expect(apiMock).toBeCalledTimes(2)
       expect(apiMock).toHaveBeenNthCalledWith(2, '/election/new', {
         method: 'POST',
-        // eslint-disable-next-line @typescript-eslint/camelcase
-        body: JSON.stringify({ organization_id: 'org-id' }),
+        body: JSON.stringify({
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          organization_id: 'org-id',
```

Comment:
Oops, should we make a follow up task to camelCase this on the backend?

Reply:
@jonahkagan I think @eventualbuddha already created a PR to do that, just need to update this to match when it gets rebased.

Reply:
Yep, this was merged in #353.

Reply:
![giphy](https://user-images.githubusercontent.com/530106/76914531-d9a3a980-6877-11ea-9d14-30b70ae18109.gif)